packages/shared/lib/services/sync/config/config.service.ts

```diff
@@ -635,8 +635,8 @@ export async function getNangoConfigIdAndLocationFromId(id: number): Promise<{ n
         .from<SyncConfig>(TABLE)
         .select(`${TABLE}.nango_config_id`, `${TABLE}.file_location`)
         .where({
-            [`${TABLE}.id`]: id,
-            [`${TABLE}.deleted`]: false
+            id,
+            deleted: false
```

Comment:
Was this throwing an error? Otherwise this change seems unrelated

Reply:
it is throwing an error: it doesn't like prefixed column like `_nango_sync_configs.id` unless there is a join